be/src/olap/cumulative_compaction.cpp

```diff
@@ -462,12 +460,12 @@ OLAPStatus CumulativeCompaction::_do_cumulative_compaction() {
     return res;
 }
 
-OLAPStatus CumulativeCompaction::_update_header(vector<SegmentGroup*>* unused_indices) {
-    vector<SegmentGroup*> new_indices;
-    new_indices.push_back(_new_segment_group);
+OLAPStatus CumulativeCompaction::_update_header(vector<RowsetSharedPtr>* unused_indices) {
+    vector<RowsetSharedPtr> new_indices;
+    new_indices.push_back(_rowset);
 
     OLAPStatus res = OLAP_SUCCESS;
-    res = _tablet->replace_data_sources(&_need_merged_versions, &new_indices, unused_indices);
+    res = _tablet->modify_rowsets(&_need_merged_versions, &new_indices, unused_indices);
```

Comment:
I think new_indices and unused_indices are not good names. pls change them

Reply:
OK